metabase/metabase
# Elevate database engines in setup

Epic: https://github.com/metabase/metabase/issues/18918

Adds special UI for the most popular database engines.

How to test:
- Run a fresh Metabase instance
- During setup it should be possible to search for and select a database
- Admin database forms should be unaffected 

<img width="727" alt="Screen Shot 2021-11-11 at 8 37 43 pm" src="https://user-images.githubusercontent.com/8542534/141343723-0ef1e163-fc70-4589-9440-8d1353140afa.png">
<img width="705" alt="Screen Shot 2021-11-11 at 8 37 52 pm" src="https://user-images.githubusercontent.com/8542534/141343733-64880c34-6710-4400-88e9-efb0e1f75b0c.png">
<img width="726" alt="Screen Shot 2021-11-11 at 8 38 01 pm" src="https://user-images.githubusercontent.com/8542534/141343735-f69e39bc-4dab-4d4a-980c-602cf9af02e2.png">
<img width="702" alt="Screen Shot 2021-11-11 at 8 39 03 pm" src="https://user-images.githubusercontent.com/8542534/141345271-e12169ec-499f-4c54-ac10-fd6c070756b8.png">
<img width="714" alt="Screen Shot 2021-11-11 at 8 38 08 pm" src="https://user-images.githubusercontent.com/8542534/141343738-522b4358-c84d-4ee0-965f-a338065c7552.png">
<img width="814" alt="Screen Shot 2021-11-11 at 9 21 50 pm" src="https://user-images.githubusercontent.com/8542534/141349463-9f040816-2219-4a01-bd0f-9bd039c2945b.png">
